orchestrators/ProcurementOperations/tests/Unit/Services/DuplicateInvoiceDetectionServiceTest.php

```
@@ -113,12 +134,43 @@ public function test_detects_invoice_number_match_with_different_amount(): void
+        $this->invoiceQuery->method('findByPOReference')->willReturn([]);
+        $this->invoiceQuery->method('findByDocumentHash')->willReturn([]);
+        $this->invoiceQuery->method('findByFingerprint')->willReturn([]);
+
+        $this->initService();
+
+        $result = $this->service->checkForDuplicates($request);
+
+        $this->assertTrue($result->hasDuplicates);
+        $this->assertEquals(DuplicateMatchType::FUZZY_INVOICE_NUMBER, $result->matches[0]->matchType);
+        $this->assertArrayHasKey('original_number', $result->matches[0]->matchDetails);
+    }
```

PHPMD (2.15.0)

[error] The method test_detects_normalized_invoice_number_match is not named in camelCase. (undefined)

(CamelCaseMethodName)